src/pkg/logger/logger.go

```diff
@@ -107,6 +135,15 @@ type Config struct {
 	Destination
 }
 
+// LogValue of config
+func (c Config) LogValue() slog.Value {
```

Comment:
When we print cfg current it looks like this
```bash
logger successfully initialized cfg={-4 console 0xc0000de068}
```
now looks like
```bash
logger successfully initialized cfg.level=debug cfg.format=console cfg.Destination=os.Stderr
```

Reply:
Good change ✨